### ISSUE TITLE
Color highlight first character of every location with first occurence of that alphabetical character for faster discovery.

### ISSUE THREAD

**Issue by chris234567** [sbsev/svelte-site#58]
The Color doesn't have to be red. Perhaps a less agressive color with good contrast still.
Alternatively we could change the sorting altogehter e.g. to lists that have alphabetical header.
I'm just not super happy with the current state.

![example](https://user-images.githubusercontent.com/59069119/174595574-89374207-6795-4854-8e3e-696b4b09ea2d.png)
